StreamChat.xcodeproj/project.pbxproj

```diff
@@ -5815,6 +5834,8 @@
 			children = (
 				C1E8AD5A278DDEBB0041B775 /* Operations */,
 				C1E8AD55278C8A440041B775 /* SyncRepository.swift */,
+				C186BFA927AA979B0099CCA6 /* SyncRepository_Tests.swift */,
+				C186BFA227A048690099CCA6 /* SyncRepository_Mock.swift */,
```

Comment:
I think that we could improve the folder structure. Right now we have something like this:

```bash
Repositories/
|——— SyncRepository.swift
|——— Operations/
|	|—— AsyncOperation.swift
|	|—— SyncOperation.swift
```

I would propose something like this:
```bash
Repositories/
 |——— SyncRepository/
 |    |——— SyncRepository.swift
 |    |——— SyncOperation.swift
```
Then the `AsyncOperation.swift` would live in the Utils or any top level Common folder (Currently the utils folder is totally crowded and there are things that shouldn't be there but anyhow..).

The reason behind this is because the Operations is an implementation detail of the `SyncRepository`. Operations can totally be used outside of the `Repositories` context, hence the `AsynOperation.swift` should live in a common place. By having an `Operations` folder in the `Repositories` folder, it gives an impression that each repository will have an operation, which is not true, since like I said before, it is an implementation detail, that is why I believe the `SyncOperatior` should live inside the `SyncRepository` context.